core/common/src/ClipStyle.ts

```diff
@@ -97,6 +98,101 @@ export class CutStyle {
   }
 }
 
+/** Wire format describing a [[ClipIntersectionStyle]].
+ * @see [[ClipStyleProps.ClipIntersectionStyle]].
+ * @public
+ * @extensions
+ */
+export interface ClipIntersectionStyleProps {
+  /** Color to apply to intersection of geometry and clip planes, default white */
+  color?: RgbColorProps;
+  /** Number of pixels to be considered intersecting the clip plane, default 1 */
+  width?: number;
+}
+
+/** As part of a [[ClipStyle]], describes how to colorize geometry intersecting the clip planes.
+ * @note Edges are highlighted only if [[ClipStyle.ClipIntersectionStyle]] is `true`.
+ * @public
+ * @extensions
+ */
+export class ClipIntersectionStyle {
+  /** Color to apply to intersection of geometry and clip planes, default white */
+  public readonly color: Readonly<RgbColor>;
```

Comment:
```suggestion
  public readonly color: RgbColor;
```
`RgbColor` is immutable; `Readonly` is redundant.